{"name": "test_forgotpassword_invalid_otp1_error", "status": "passed", "attachments": [{"name": "stdout", "source": "923d2069-ff23-4720-b546-b6f517f35566-attachment.txt", "type": "text/plain"}], "start": 1757351582672, "stop": 1757351583537, "uuid": "3420c78e-1665-4e0e-b0ec-1ed246316c63", "historyId": "6ec9ad9310dcc6217263c941384037c4", "testCaseId": "6ec9ad9310dcc6217263c941384037c4", "fullName": "tests.test_forgotpassword#test_forgotpassword_invalid_otp1_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_forgotpassword"}, {"name": "host", "value": "OTSI-Lap80"}, {"name": "thread", "value": "60208-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_forgotpassword"}]}